### PR TITLE
fix two digit trailing zeros < 0.10

### PR DESCRIPTION
**Pull request by paulbjarne**
Fixes two place decimal formatting after zero for < 0.10

### CODE REVIEW

File: src/Data/Formatter/Number.purs [hunk 1]
```diff
@@ -90,7 +90,9 @@ format ∷ Formatter → Number → String
 format (Formatter f) num =
   let
     absed = Math.abs num
-    tens = if absed > 0.0 then Int.floor $ Math.log absed / Math.ln10 else 0
+    tens = if absed > 0.0
+             then max (Int.floor $ Math.log absed / Math.ln10) 0
```
Comment (safareli) [inline src/Data/Formatter/Number.purs]: So the issue was that `Math.log` for anything less then 1.0, will return negative number

File: src/Data/Formatter/Number.purs [hunk 2]
```diff
@@ -118,6 +120,11 @@ format (Formatter f) num =
            let
              multiplier = Math.pow 10.0 $ Int.toNumber f.after
            in Int.round $ leftover * multiplier
+         roundedWithZeros =
```
Comment (safareli) [inline src/Data/Formatter/Number.purs]: I don't understand this part, @cryogenian can you take a look?

Comment (cryogenian) [reply src/Data/Formatter/Number.purs]: This adds zeros after comma/point and digits.

**Comment by cryogenian** [comment]
@paulbjarne Thank you!

**Pull request merged**